### PR TITLE
Updated Control Center design #715

### LINKED ISSUE #715: Update Control Center design
TODO: 
- [ ] Reorganize the bottom section to horizontal layout (similar to #713) 
- [ ] Update What's New and Home and Privacy icons

IMG_5357

Icons to update:
[whats-new-24.pdf](https://github.com/cliqz/user-agent-ios/files/4097811/whats-new-24.pdf)
[Home24.pdf](https://github.com/cliqz/user-agent-ios/files/4097812/Home24.pdf)
[About-Privacy24.pdf](https://github.com/cliqz/user-agent-ios/files/4097813/About-Privacy24.pdf)

New design:
image (2)
image

### PR DESCRIPTION
**Pull request by shahenAnt**
Fix #715 

## Implementation details
Added new collection cell in control center.
Screen Shot 2020-02-10 at 17 31 57
Simulator Screen Shot - iPad (7th) iOS 13 3 - 2020-02-10 at 17 32 28

## Checklist:

- [ ] My squashed commit messages follows the [7 golden rules](https://chris.beams.io/posts/git-commit/)
- [ ] I updated or created necessary unit tests
- [ ] I've tested my changes in Dark Mode, iPad, iOS <= 12, Landscape and in German

### CODE REVIEW

File: Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift [hunk 1]
```diff
@@ -432,6 +430,15 @@ extension PhotonActionSheetProtocol {
         return openDownloads
     }
 
+    private func collectionItem(vcDelegate: PageOptionsVC, openHomePage: PhotonActionSheetItem) -> PhotonActionSheetItem {
```
Comment (chrmod) [inline Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift]: I would rather not have this function at all as it is being used only once and it's interface is confusing. Could you please move the implementsation directly to the where it is being used?

File: Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift [hunk 2]
```diff
@@ -394,3 +406,16 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
         return view
     }
 }
+
+extension PhotonActionSheet: PhotonActionSheetCollectionCellDelegate {
+
+    func collectionCellDidSelectItem(item: PhotonActionSheetItem) {
+        guard let handler = item.handler else {
+            self.dismiss(nil)
```
Comment (chrmod) [inline Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift]: dismiss is called in all cases so can be moved out of guard

Comment (shahenAnt) [reply Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift]: Done.

File: Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift (hunk 1 above)
Comment (shahenAnt) [reply Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift]: Done.

**Pull request merged**